src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -35,14 +37,12 @@ public function __construct(ManagerRegistry $registry)
     }
 
     /**
-     * @param object $entity
-     *
      * @return void
      *
      * @throws UnexpectedTypeException
      * @throws ConstraintDefinitionException
      */
-    public function validate(mixed $entity, Constraint $constraint)
+    public function validate(mixed $value, Constraint $constraint)
```

Comment:
Maybe `$object` would be more clear?

Reply:
There was $object in the past, but Psalm had a problem with it https://github.com/symfony/symfony/pull/38662/commits/dca83225a6ea105938ebb8ac3a5ff56a54cead12